docroot/profiles/custom/cgov_site/themes/custom/cgov/static/images/design-elements/icons/favicons/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/profiles/custom/cgov_site/themes/custom/cgov/static/images/design-elements/icons/favicons/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/profiles/custom/cgov_site/themes/custom/cgov/static/images/design-elements/icons/favicons/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
```

Comment:
Change the filenames to `favicon-192.png` and `favicon-512.png`. And, of course, make sure the actual filenames match.